lib/comparing.dart

```diff
@@ -26,7 +26,8 @@ class Comparing {
 
   /// Join several mappers and compare in order (first the first element, then
   /// the second, and so on).
-  static Comparator<T> join<T>(List<Comparable Function(T t)> mappers) {
+  static Comparator<T> join<T>(
+      List<Comparable<Object?> Function(T t)> mappers) {
```

Comment:
missing trailing comma

Reply:
I wonder why the pipeline didn't pick this up

Reply:
exactly my thoughts, I will take a look